src/azure-cli/azure/cli/command_modules/acr/task.py

```diff
@@ -53,7 +54,7 @@ def acr_task_create(cmd,  # pylint: disable=too-many-locals
                     status='Enabled',
                     platform=None,
                     cpu=DEFAULT_CPU,
-                    timeout=DEFAULT_TIMEOUT_IN_SEC,
+                    timeout=ACR_RUN_DEFAULT_TIMEOUT_IN_SEC,
```

Comment:
can you double-check in the parameters file that this is also the value displayed for the command?